apps/explorer/src/app/components/txs/txs-infinite-list-item.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import {
+  Dialog,
+  Icon,
+  Intent,
+  SyntaxHighlighter,
+} from '@vegaprotocol/ui-toolkit';
+import { TruncatedLink } from '../truncate/truncated-link';
+import { Routes } from '../../routes/route-names';
+import { TxOrderType } from './tx-order-type';
+import type { ChainExplorerTxResponse } from '../../routes/types/chain-explorer-response';
+
+const truncateLength = 14;
```

Comment:
nitpick (non-blocking): 
```suggestion
const TRUNCATE_LENGTH = 14;
```